Dockerfile

```diff
@@ -2,8 +2,7 @@ FROM nvcr.io/nvidia/pytorch:24.01-py3
 ARG DEBIAN_FRONTEND=noninteractive
 WORKDIR /openduck-py
 
-# Copying is only necessary on prod, when you're not mounting the volume
-# COPY ./openduck-py /openduck-py
+COPY ./openduck-py /openduck-py
```

Comment:
this needed to come back to avoid problems, for example requirements.txt wasn't being copied into the docker container